projects/core/src/product/occ/converters/product-image-normalizer.ts

```diff
@@ -1,12 +1,25 @@
 import { Injectable } from '@angular/core';
 
-import { OccConfig } from '../../../occ/index';
+import { OccConfig } from '../../../occ';
 import { Product } from '../../../occ/occ-models/occ.models';
+import { Converter } from '../../../util/converter.service';
 
-@Injectable()
-export class ProductImageConverterService {
+@Injectable({
+  providedIn: 'root',
+})
+export class ProductImageNormalizer implements Converter<Product, Product> {
```

Comment:
Can we add the UI model here? The backend model has a single list of images, while we like to have a more semantic structure in the UI model. We do not have this reflected in the UI model though.